src/rbd.cc

```diff
@@ -499,6 +475,7 @@ static int do_clone(librbd::RBD &rbd, librados::IoCtx &p_ioctx,
   if ((features & RBD_FEATURE_LAYERING) != RBD_FEATURE_LAYERING) {
     return -EINVAL;
   }
+  features &= ~RBD_FEATURE_STRIPINGV2;
 
   return rbd.clone(p_ioctx, p_name, p_snapname, c_ioctx, c_name, features,
```

Comment:
It was breaking CLI integration test since before the change to rbd_default_features, the striping bit would not be enabled for clones (effectively the behavior I was reverting to here).  If I swap out  this commit for one that uses clone2 and the user specified options, all will be good.